cmd/createcluster.go

```diff
@@ -405,22 +403,38 @@ func getValidators(dvs []tbls.TSS) ([]cluster.DistValidator, error) {
 func writeKeysToKeymanager(ctx context.Context, addrs []string, numNodes int, shareSets [][]*bls_sig.SecretKeyShare) error {
 	// Ping all keymanager addresses to check if they are accessible to avoid partial writes
 	for i := 0; i < numNodes; i++ {
-		if err := verifyConnection(ctx, addrs[i]); err != nil {
+		cl := keymanager.New(addrs[i])
+		if err := cl.VerifyConnection(ctx); err != nil {
 			return err
 		}
 	}
 
 	for i := 0; i < numNodes; i++ {
-		var secrets []*bls_sig.SecretKey
+		var (
+			keystores []keystore.Keystore
+			passwords []string
+		)
 		for _, shares := range shareSets {
+			password, err := testutil.RandomHex32()
+			if err != nil {
+				return err
+			}
+			passwords = append(passwords, password)
+
 			secret, err := tblsconv.ShareToSecret(shares[i])
 			if err != nil {
 				return err
 			}
-			secrets = append(secrets, secret)
+
+			store, err := keystore.Encrypt(secret, password, rand.Reader)
+			if err != nil {
+				return err
+			}
+			keystores = append(keystores, store)
 		}
 
-		err := postKeysToKeymanager(ctx, addrs[i], secrets)
+		cl := keymanager.New(addrs[i])
+		err := cl.ImportKeystores(ctx, keystores, passwords)
```

Comment:
suggest logging info level: `Imported key shares to keymanger, node=node0, keymanger-address=addr[i]`